cdk/lib/amiable/amiable.ts

```diff
@@ -38,7 +37,17 @@ export class Amiable extends GuStack {
       certificateProps: {
         domainName: props.domainName,
       },
-      monitoringConfiguration: { noMonitoring: true },
+      monitoringConfiguration:
+        this.stage === "PROD"
+          ? {
+              snsTopicName: "devx-alerts",
+              http5xxAlarm: {
+                tolerated5xxPercentage: 99,
```

Comment:
It's really hard to tune alarms for low-traffic services like this. Because we are using % based alerting, it's quite possible that 1 or 2 requests resulting in a 5XX response would lead to a high % of overall requests being 5XXs. Perhaps we need to think about supporting an alarm based on absolute number of requests for services like this...

For now, I have decided to attempt to catch the relatively severe cases only e.g. Amiable is completely down for 2 minutes (or more) or Amiable is up but there are 2 consecutive minutes where all responses are 5XXs.